### DISCUSSION TITLE
I have zero clue why it says Tweakscale is not installed correctly

### DISCUSSION THREAD

**Discussion by Alexsys** [TweakScale/TweakScale#299] [Support] [answered]
https://i.imgur.com/3kCcPM1.png

https://gist.github.com/Alexsys/b8702ac42e14d7b7ef77e662739a9a98

https://gist.github.com/Alexsys/c26b47568c3d8d1ed6aa1ac7afff250a

@Lisias
sup man, its time to return to ksp again, yet the bull fights and doesnt want me to game.
I just reinstalled all the mods, fresh, deleting em individually and replacing with the newest versions.
Game acts up.

I love you

**Comment by Lisias** [comment, marked as answer]
Hi, @Alexsys !

You got bitten by the infamous KSP bug on the `Assembly Loader/Resolver` where anything going wrong while loading **any** DLL screws up the `AL/R` and then all further DLL loading get screwed.

In your case, the culprit is…

```
[EXC 21:36:34.159] Add to mod list threw an exception in loading ScrapYard_ContractConfigurator, Version=2.2.99.1, Culture=neutral, PublicKeyToken=null: System.Reflection.ReflectionTypeLoadException: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at ModuleManager.ModListGenerator.GetAdditionalModsFromStaticMethods (ModuleManager.Logging.IBasicLogger logger) [0x00021] in <ac37db06097f4706b70066f555f86981>:0 
```

Delete `<YOUR-KSP-ROOT>/GameData/ScrapYard/Plugins/ScrapYard_ContractConfigurator.dll` and you will be fine!

**Reply by zer0Kerbal** [reply]
yes @Alexsys KPS 1.8.1 (IIRC) made this an issue - that is why there are two versions of each release now on [CurseForge](https://www.curseforge.com/kerbal/ksp-mods/scrapyard/files) with the `N`version not including the CC.dll.

Thank you @Lisias

**Comment by Alexsys** [comment]
NO WAY it's this same issue from IONSSSSSSSSSS ago.

I'll give it a try, if it is.

SCRAPYARD U R UNBELIEVABLE

**Reply by zer0Kerbal** [reply]
> SCRAPYARD U R UNBELIEVABLE

actual fault lies with KSP

**Comment by Alexsys** [comment]
Yep, all is well.

Thank you for the quick reply Mister Lisias.

**Comment by Alexsys** [comment]
KSP U R UNBELIEVABLE



Thats better :)
